### PR TITLE
feat(event): add cow event page

### PR DESCRIPTION
**Pull request by DaveBitter**
Add custom event page for the CoW meetup in amsterdam

### CODE REVIEW

File: pages/events/cow-ams-frontend-frameworks-2023.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            <RaffleBlock href="https://techhub.iodigital.com" numberOfTickets={2}>
+              JSNation Amsterdam
+              <br />
+              Raffle
+            </RaffleBlock>
+            <RaffleBlock href="https://techhub.iodigital.com" numberOfTickets={1}>
+              Vue.js London
+              <br />
+              Raffle
+            </RaffleBlock>
```
Comment (DaveBitter) [inline pages/events/cow-ams-frontend-frameworks-2023.js:79]: Add correct raffle links

File: pages/events/cow-ams-frontend-frameworks-2023.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            Win tickets
+            <br />x{numberOfTickets}
+          </p>
+        </div>
+      </div>
+    </div>
+  )
+}
+
+export default function Experiments() {
```
Comment (sanderdejong88) [inline pages/events/cow-ams-frontend-frameworks-2023.js]: Experiments?

File: pages/events/cow-ams-frontend-frameworks-2023.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  const { theme } = useBrandingTheme()
+
+  return (
+    <>
+      <PageSEO
+        title="Events - Coven of Wisdom Amsterdam - Frontend Frameworks - 2023"
+        description="What is going on with all these new frameworks like Astro, Qwik, RedwoodJS and SolidJS?"
```
Comment (sanderdejong88) [inline pages/events/cow-ams-frontend-frameworks-2023.js]: Add RSC

File: pages/events/cow-ams-frontend-frameworks-2023.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+          <header className="flex flex-col gap-4 text-center text-white ">
+            <h1
+              className={`bg-io_${theme}-900 rounded-full py-2 px-4 font-serif text-3xl font-light`}
+            >
+              Coven of Wisdom
+            </h1>
+            <p>Frontend Frameworks 2023</p>
+            <img
+              src="/images/cow/cow-mascotte.png"
+              alt=""
```
Comment (sanderdejong88) [inline pages/events/cow-ams-frontend-frameworks-2023.js]: Alt?

File: pages/events/cow-ams-frontend-frameworks-2023.js (hunk 1 above)
Comment (DaveBitter) [reply pages/events/cow-ams-frontend-frameworks-2023.js:79]: We'll do this later

**Pull request merged**